setup.cfg

```diff
@@ -1,12 +1,22 @@
 [flake8]
-exclude = .git,.tox,__pycache__,env,venv,build,dist,docs
+exclude = .git,.tox,__pycache__,env,venv,build,dist,docs,splunk_eventgen/lib/concurrent/futures/__init__.py
```

Comment:
The correct third party lib folders to exclude are: `splunk_eventgen/lib/concurrent/futures` and `splunk_eventgen/lib/requests_futures`. right?

Reply:
Correct, I'll include 'requests_futures' as well.

Reply:
You may also need to revert the changes to the files under these two folders.